DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/divider.js

```diff
@@ -56,15 +56,11 @@
 						cssproperties: [
 							{ name: "width" }
 						],
-						unitOptions: true,
-						units: [
-                            { name: 'px' },
-                            { name: '%' }
-						],
+                        units: [
+                            { name: 'px', min: 10, max: 1000, step: 1, value: 100 },
```

Comment:
Why max 1000? Can't I create a divider that's 1980 px wide ?

Reply:
Updated divider max to 1980 px